### PR TITLE
Support for ghq managed repositories

### PR DESCRIPTION
**Pull request by uetchy**
This PR gives combined view for $GOPATH and `ghq root`.
Also it includes a bug fix for walking directories.

``` session
❯ gch
/Users/uetchy/.go/src/github.com/b4b4r07/gch
 M cli.go
 M walk.go
/Users/uetchy/.ghq/github.com/uetchy/ferret
A  .eslintrc
M  .gitignore
A  app/ArticleList.jsx
A  app/Header.jsx
A  app/app.jsx
M  index.css
M  index.html
M  index.js
M  package.json
```

Could you please merge this?

### CODE REVIEW

File: cli.go [hunk 1]
```diff
@@ -36,15 +37,28 @@ func (cli *CLI) Run(args []string) int {
 	cpu := runtime.NumCPU()
 	runtime.GOMAXPROCS(cpu)
 
-	for _, gp := range filepath.SplitList(os.Getenv("GOPATH")) {
```
Comment (babarot) [inline cli.go:39]: Thank you for Pull Request.
Let me make sure I understand you correctly. You mean, `gch` will browse `$GOPATH` and `ghq root`. Is that right?
I think it's a very good idea. However, your changes does not correspond multiple `$GOPATH`.

Comment (uetchy) [inline cli.go:39]: I'm surprised to know $GOPATH allows to have one or more paths :open_mouth: .
Let me fix them.

**Comment by babarot** [comment]
Thank you for your fix.
BTY, what kind of situation are the use case that the `ghq root` and `$GOPATH` are different?
I don't know because they are the same in my case.

**Comment by uetchy** [comment]
I use **$GOPATH** as a warehouse only for Go based projects and often use `ghq` for non-Go based projects; that's why I need this feature.

**Comment by babarot** [comment]
Oh, now I get it. I'll merge that.

**Pull request merged**